cmd/proxy-server/main.go

```diff
@@ -25,18 +25,19 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-logr/logr"
+	"github.com/go-logr/stdr"
```

Comment:
if we're going to be grouping make the logr one a separate group too

Reply:
It's a 3rd party dependency so I'm curious to hear why we would separate it into it's own group? I've been strictly following https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e